project_reach/lib/dataset_specification.py

```diff
@@ -0,0 +1,32 @@
+from core_data_modules.cleaners import somali
+
+
+class CodingPlan(object):
```

Comment:
Is this a REACH specific data structure? Seems like it could be more general?

Reply:
It is for REACH and very-very-similar-to-REACH projects. As currently implemented, it won't work if we need multiple code schemes for demographic/follow-up questions, as we did on Wellcome.